test/support/noapi_controller.ex

```diff
@@ -0,0 +1,15 @@
+defmodule OpenApiSpexTest.NoApiController do
+  @moduledoc false
+
+  use Phoenix.Controller
+  use OpenApiSpex.ControllerSpecs
+
+  plug OpenApiSpex.Plug.CastAndValidate, json_render_error_v2: true
```

Comment:
@rolandtritsch I suggest adding a test for the struct operation format to ensure it works in that case as well.
See: https://github.com/open-api-spex/open_api_spex/blob/master/examples/phoenix_app/lib/phoenix_app_web/controllers/user_controller_with_struct_specs.ex

Reply:
Added. Fixed.